openedx/core/djangoapps/notifications/views.py

```diff
@@ -284,9 +285,9 @@ def get(self, request):
         # Get the unseen notifications count for each app name.
         count_by_app_name = (
             Notification.objects
-            .filter(user_id=request.user, last_seen__isnull=True)
+            .filter(user_id=request.user)
             .values('app_name')
-            .annotate(count=Count('*'))
+            .annotate(count=Coalesce(Count('id', filter=Q(last_seen__isnull=True)), 0))
```

Comment:
I am unsure what this function does,
```
 .annotate(count=Coalesce(Count('id', filter=Q(last_seen__isnull=True)), 0))
```
 please explain it a bit, i.e. expected response. and please check why unit tests didn't fail after this change.